fpdf/table.py

```diff
@@ -32,6 +33,7 @@ def __init__(
         gutter_height=0,
         gutter_width=0,
         headings_style=DEFAULT_HEADINGS_STYLE,
+        index_style=DEFAULT_INDEX_STYLE,
```

Comment:
New parameters should always be added at the end of the parameters list, otherwise the code of existing `fpdf2` users would break if they are currently passing parameters by value, up to `line_height` for example there!
```python
align = "CENTER"
v_align = "MIDDLE",
borders_layout = TableBordersLayout.ALL
cell_fill_color = None
cell_fill_mode = TableCellFillMode.NONE
col_widths = None
first_row_as_headings = True
gutter_height = 0
gutter_width = 0
headings_style = DEFAULT_HEADINGS_STYLE
line_height = None

with pdf.table(align, v_align, borders_layout, cell_fill_color, cell_fill_mode, col_widths, first_row_as_headings, gutter_height, gutter_width, headings_style, line_height) as table:
    ...  # this code would break after merging this PR, because line_height would be passed to index_style
```